formats/protobuf/commonTest/src/kotlinx/serialization/protobuf/schema/SchemaValidationsTest.kt

```diff
@@ -39,6 +39,33 @@ class SchemaValidationsTest {
         SECOND
     }
 
+    @Serializable
```

Comment:
Please add this class only negative case: e.q. `assertFailsWith` for 
```
    enum class EnumWithExplicitProtoNumberDuplicate {
        @ProtoNumber(2)
        FIRST,
        @ProtoNumber(2)
        SECOND,
    }
```
and/or 
```
    enum class EnumWithImplicitProtoNumberDuplicate {
        FIRST,
        @ProtoNumber(0)
        SECOND,
    }
```
and positive cases in `kotlinx.serialization.protobuf.schema.GenerationTest` similar to the existing ones.